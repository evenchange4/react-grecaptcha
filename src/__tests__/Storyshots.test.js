import initStoryshots from '@storybook/addon-storyshots';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

const styledSnapshot = ({ story, context }) => {
  const storyElement = story.render(context);
  const tree = mount(storyElement);
  expect(toJson(tree)).toMatchSnapshot();
};

initStoryshots({
  test: styledSnapshot,
});
