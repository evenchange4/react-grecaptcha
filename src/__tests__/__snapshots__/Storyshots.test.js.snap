// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Recaptcha Invisible reCAPTCHA 1`] = `
<Recaptcha
  callback={[Function]}
  expiredCallback={[Function]}
  invisible={true}
  locale="zh-TW"
  sitekey="6LdCeh0UAAAAAK8d4SISsIi7KcykpnP7x9D2vlha"
>
  <div
    className="g-recaptcha"
    data-callback="_grecaptcha.data-callback"
    data-expired-callback="_grecaptcha.data-expired-callback"
    data-sitekey="6LdCeh0UAAAAAK8d4SISsIi7KcykpnP7x9D2vlha"
    data-size="invisible"
  />
</Recaptcha>
`;

exports[`Storyshots Recaptcha With locale 1`] = `
<Recaptcha
  callback={[Function]}
  expiredCallback={[Function]}
  invisible={false}
  locale="zh-TW"
  sitekey="6LehviATAAAAACZ5hcTODQmVldaS5fVHAOKbw3MP"
>
  <div
    className="g-recaptcha"
    data-callback="_grecaptcha.data-callback"
    data-expired-callback="_grecaptcha.data-expired-callback"
    data-sitekey="6LehviATAAAAACZ5hcTODQmVldaS5fVHAOKbw3MP"
  />
</Recaptcha>
`;

exports[`Storyshots Recaptcha With otherProps 1`] = `
<Recaptcha
  callback={[Function]}
  data-theme="dark"
  expiredCallback={[Function]}
  invisible={false}
  locale="en"
  sitekey="6LehviATAAAAACZ5hcTODQmVldaS5fVHAOKbw3MP"
>
  <div
    className="g-recaptcha"
    data-callback="_grecaptcha.data-callback"
    data-expired-callback="_grecaptcha.data-expired-callback"
    data-sitekey="6LehviATAAAAACZ5hcTODQmVldaS5fVHAOKbw3MP"
    data-theme="dark"
  />
</Recaptcha>
`;

exports[`Storyshots Recaptcha reCAPTCHA v2 1`] = `
<Recaptcha
  callback={[Function]}
  expiredCallback={[Function]}
  invisible={false}
  locale="en"
  sitekey="6LehviATAAAAACZ5hcTODQmVldaS5fVHAOKbw3MP"
>
  <div
    className="g-recaptcha"
    data-callback="_grecaptcha.data-callback"
    data-expired-callback="_grecaptcha.data-expired-callback"
    data-sitekey="6LehviATAAAAACZ5hcTODQmVldaS5fVHAOKbw3MP"
  />
</Recaptcha>
`;
